### PR TITLE
Products

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,85 @@
...
 
 function App() {
+  const { cartState } = useCart();
+
+  const { wishlistState } = useWishlist();
+
+  const wishItemNum = wishlistState.length;
+
+  // const num = wishlistState.length
+  // const {num} = useWishlist();
```
Comment (sks-21) [inline src/App.js:24]: remove commented code

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,85 @@
...
+                </span>
+              </i>
+            </Link>
+
+            {/* notification badge  */}
+
+            {/* <Link  to="/CartPage" ><i className=" notification nav-icons fas fa-shopping-cart "> <span className="badge">3</span></i>  </Link> */}
+          </span>
+
+          {/* <Link to ="Product">Product </Link> */}
```
Comment (sks-21) [inline src/App.js:71]: Same as above

File: src/components/CartPage/CartPage.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import CartItem from "../list/CartItem";
+
+function CartPage() {
+  return (
+    <div>
+      {/* <h1> This is Cart Page</h1> */}
```
Comment (sks-21) [inline src/components/CartPage/CartPage.js:7]: same as above

File: src/components/HomePage/Home.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+        <Link className="text-on-image" to="/Product">
+          <h1> Explore ! </h1>
+        </Link>
+
+        <Routes>
+          <Route path="/Product" element={<Product />} />
+        </Routes>
+      </div>
+      <br /> <br />
```
Comment (sks-21) [inline src/components/HomePage/Home.js:20]: You can make a utility class for providing vertical spacing.

File: src/components/HomePage/Home.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      <h1 className="heading">Top Categories</h1>
+      <br></br> <br />
+      <div className="cards-container">
+        <div className="card">
+          <img className="card-image" src="../images/imac/imac1.jpg" alt="" />
+
+          {/* <div className="top-icon-red "> 
+                  <i className="fas fa-heart card-heart-icon" > </i> 
+              </div> */}
+
```
Comment (sks-21) [inline src/components/HomePage/Home.js:30]: remove commented code.

File: src/components/HomePage/Home.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+          </div>
+        </div>
+
+        <div className="card">
+          <img className="card-image" src="../images/ipad/ipad1.jpg" alt="" />
+
+          {/* <div className="top-icon-red "> 
+                  <i className="fas fa-heart card-heart-icon" > </i> 
+              </div> */}
+
```
Comment (sks-21) [inline src/components/HomePage/Home.js:46]: same

File: src/components/HomePage/Home.js [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+          <img
+            className="card-image"
+            src="../images/watch/apple-watch2.jpg"
+            alt=""
+          />
+
+          {/* <div className="top-icon-red "> 
+                  <i className="fas fa-heart card-heart-icon" > </i> 
+              </div> */}
+
```
Comment (sks-21) [inline src/components/HomePage/Home.js:66]: same as above

File: src/components/HomePage/home.css [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+.cards-container {
+  display: flex;
+  /* justify-content: center; */
+  gap: 4rem;
+}
+
+.card {
+  position: relative;
+  font-family: monospace;
+  width: 260px;
```
Comment (sks-21) [inline src/components/HomePage/home.css:43]: you may use rems

File: src/components/HomePage/navbar.css [hunk 9]
```diff
@@ -0,0 +1,218 @@
...
+}
+
+.btn {
+  font-size: 1.1rem;
+  background-color: var(--primary-color);
+  color: var(--text-color);
+  margin-top: 0.4rem;
+  margin-left: 0.8rem;
+  border-style: none;
+  padding: 0.3rem;
```
Comment (sks-21) [inline src/components/HomePage/navbar.css:54]: for values, less than 10px do not use rems prefer using pixels.

File: src/components/HomePage/navbar.css [hunk 10]
```diff
@@ -0,0 +1,218 @@
...
+}
+
+.btn-login {
+  background-color: var(--text-color);
+  color: var(--primary-color);
+  font-weight: 500;
+  border-radius: 0.2rem;
+  padding: 0.3rem 1.3rem;
```
Comment (sks-21) [inline src/components/HomePage/navbar.css:62]: same as above

**Review by sks-21 — APPROVED** [review]
I would suggest you make a separate branch for separate purposes.
There are a lot of file changes which makes reviewing difficult.

Rest everything looks nice. Go through the comments once.

**Review by manojkumar8400 — APPROVED** [review]

**Pull request merged**